lib/src/metta/text.rs

```diff
@@ -253,10 +266,11 @@ mod tests {
     }
 
     #[test]
+    #[ignore]
```

Comment:
We could just check that a proper error is returned instead of expecting a panic. It also makes sense to add unit test for other possible errors.